template.rb

```diff
@@ -33,10 +33,10 @@ def render_file(filename)
   end
 end
 
-skip_active_admin = false
-skip_devise = false
-# skip_active_admin = yes?("Skip ActiveAdmin?")
-# skip_devise = yes?("Skip Devise?")
+# skip_active_admin = false
+# skip_devise = false
+skip_active_admin = yes?("Skip ActiveAdmin?")
```

Comment:
(This can become a separate card/PR)

I think we can now assume that we're including ActiveAdmin on every project, so these `ask`s and the conditional checks that they support can be safely removed.

Reply:
Oops. I didn't realize the `yes?` method had been commented out in `master`. I think that was a mistake after testing so long. I'll remove this change here and make a small separate pr for it.